frontend/src/metabase/query_builder/components/expressions/ExpressionEditorTextfield/ExpressionEditorTextfield.tsx

```diff
@@ -392,6 +452,8 @@ class ExpressionEditorTextfield extends React.Component {
     },
     {
       name: "chooseSuggestion",
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore // Based on typings null is not a valid value, but bindKey is assigned dynamically if there are suggestions available.
```

Comment:
Could we augment ace types to make it allowed? The same as previous comment, ideally in one place in `frontend/src/types`, where other augmented types are